.github/workflows/linux_builds.yml

```
@@ -68,6 +68,66 @@ jobs:
         run: |
           ./bin/godot.linuxbsd.opt.tools.64.mono --test
 
+  linux-editor-sanitizers:
+    runs-on: "ubuntu-20.04"
+    name: Editor with sanitizers (target=debug, tools=yes, tests=yes, use_asan=yes, use_ubsan=yes)
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Azure repositories are not reliable, we need to prevent azure giving us packages.
+      - name: Make apt sources.list use the default Ubuntu repositories
+        run: |
+          sudo cp -f misc/ci/sources.list /etc/apt/sources.list
```

```suggestion
          sudo rm -f /etc/apt/sources.list.d/*
          sudo cp -f misc/ci/sources.list /etc/apt/sources.list
```
(See cf2e0116d9a0ff09bba0553fe7b86f48fa82a23b.)